src/ape/managers/converters.py

```diff
@@ -70,6 +70,18 @@ def convert(self, value: Any) -> int:
         return to_int(HexBytes(value))
 
 
+class HexListConverter(ConverterAPI):
+    """
+    Convert list of hex values to single concatenated ``HexBytes`` value.
+    """
+
+    def is_convertible(self, value: Any) -> bool:
+        return isinstance(value, list) and all(is_hex(v) or isinstance(v, bytes) for v in value)
```

Comment:
it might also be nice to support tuples here

```suggestion
        return isinstance(value,(list, tuple)) and all(is_hex(v) or isinstance(v, bytes) for v in value)
```

Reply:
I wonder if there's a way to just be like, generally, this supports the iterable protocol.

Reply:
there is!

```python
>>> from collections.abc import Iterable
>>> isinstance([1,2,3], Iterable)
True
>>> isinstance((1,2,3), Iterable)
True
```